python/smqtk/tests/algorithms/relevancy_index/test_svm_hik.py

```diff
@@ -103,3 +92,20 @@ def test_simple_iqr_scenario(self):
             ntools.assert_equal(rank_ordered[4][0], self.d6)
             ntools.assert_equal(rank_ordered[5][0], self.d3)
             ntools.assert_equal(rank_ordered[6][0], self.d4)
+
+        def test_simple_iqr_scenario(self):
+            # Make some descriptors;
+            # Pick some from created set that are close to each other and use as
+            #   positive query, picking some other random descriptors as
+            #   negative examples.
+            # Rank index based on chosen pos/neg
+            # Check that positive choices are at the top of the ranking (closest
+            #   to 0) and negative choices are closest to the bottom.
+            iqr_index = LibSvmHikRelevancyIndex()
+            iqr_index.build_index(self.index_descriptors)
+
+            rank = iqr_index.rank([self.q_pos], [self.q_neg])
+            self._test_simple_iqr_helper(rank)
+            rank_libsvm = iqr_index.rank([self.q_pos], [self.q_neg],
+                                         {"use_libsvm": True})
+            self._test_simple_iqr_helper(rank_libsvm)
```

Comment:
Should there be an assert that the two rank results are equal?  I ask because of the TODO you wrote above.

Reply:
Yeah, I believe calling both `self._test_simple_iqr_helper(rank)` and  `self._test_simple_iqr_helper(rank_libsvm)` takes care of that?